control_plane/src/background_process.rs

```diff
@@ -36,8 +36,8 @@ use utils::pid_file::{self, PidFileRead};
 // it's waiting. If the process hasn't started/stopped after 5 seconds,
 // it prints a notice that it's taking long, but keeps waiting.
 //
-const RETRY_UNTIL_SECS: u64 = 10;
-const RETRIES: u64 = (RETRY_UNTIL_SECS * 1000) / RETRY_INTERVAL_MILLIS;
+const DEFAULT_RETRY_TIMEOUT_SECS: u64 = 10;
```

Comment:
This seems like yet another duplicate of the default 10s.

Reply:
This is a default for stopping the servers